src/components/Navigation/index.js

```diff
@@ -38,146 +43,166 @@ export default class Navigation extends Component {
+                      <NavigationLinkContents />
+                    </a>
+                  )
+                }
+
+                else {
+                  return startsWith(item.action, '/')
+                    ? <NavLink
+                        exact
+                        key={index}
+                        className={sharedLinkClassnames}
+                        activeClassName={activeLinkClassnames}
+                        activeStyle={activeLinkStyle}
+                        style={sharedLinkStyle}
+                        onClick={() => {
+                          if(isMobile) {
+                            this.close()
+                          }
+                        }}
+                        to={item.action}
+                      >
+                        <NavigationLinkContents />
+                      </NavLink>
+                    : <a
+                        key={index}
+                        className={sharedLinkClassnames}
+                        style={sharedLinkStyle}
+                        onClick={() => {
+                          if(isMobile) {
+                            this.close()
+                          }
+                        }}
+                        href={item.action}
+                      >
+                        <NavigationLinkContents />
+                      </a>
+                }
+
+              })}
+            </nav>
+
+            <div className='mt4 tc'>
+              <Link to={'/lessons/new'}>
+                <Button 
+                  size='extra-small'
+                  color='green'
                 >
-                  {item.text}
-                </a>
-              )
-            }
-
-            else {
-              return startsWith(item.action, '/')
-                ? <NavLink
-                    exact
-                    key={index}
-                    className={sharedLinkClassnames}
-                    activeClassName={activeLinkClassnames}
-                    style={sharedLinkStyle}
-                    onClick={() => {
-                      if(isMobile) {
-                        this.close()
-                      }
-                    }}
-                    to={item.action}
-                  >
-                    {item.text}
-                  </NavLink>
-                : <a
-                    key={index}
-                    className={sharedLinkClassnames}
-                    style={sharedLinkStyle}
-                    onClick={() => {
-                      if(isMobile) {
-                        this.close()
-                      }
-                    }}
-                    href={item.action}
-                  >
-                    {item.text}
-                  </a>
-            }
-
-          })}
-        </nav>
-
-      </aside>
+                  <Text message='navigation.action' />
+                </Button>
+              </Link>
+            </div>
+
+          </aside>
+        )}
+      </DeviceWidth>
     )
```

Comment:
Updated navigation styles:
![image](https://cloud.githubusercontent.com/assets/5497885/24430035/f1361478-13d1-11e7-9a77-81e179bf8227.png)

Not pixel perfect to mocks, but pretty close using tachyons-egghead. The button styles will be updated in: 
https://github.com/eggheadio/egghead-ui/issues/93